ghostly.py

```diff
@@ -162,6 +172,29 @@ def switchTo(self, selector):
         self.browser.switch_to_frame(selector)
 
 
+    def assertElement(self, selector):
+        """
+        Assert that an element exists on currently displayed page.
+        """
+        self.wait(1)
```

Comment:
Let's not put waits in the asserts - we want the user to be able to specify their wait times themselves if it's necessary